packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -737,6 +737,136 @@ export class ContainerRuntime extends TypedEventEmitter<IContainerRuntimeEvents>
         return runtime;
     }
 
+    /**
+     * Load the stores from a snapshot and returns the runtime.
+     * @param context - Context of the container.
+     * @param registryEntries - Mapping to the stores.
+     * @param existing - When loading from an existing snapshot
+     * @param requestHandler - Request handlers for the container runtime
+     * @param runtimeOptions - Additional options to be passed to the runtime
+     */
+     public static async loadRuntime(
```

Comment:
Can you just call the other one from here, or vice-versa?  Not ideal to duplicate all this code.

Reply:
And aren't you going to mark the other one as deprecated?